sklearn/manifold/_locally_linear.py

```diff
@@ -20,6 +20,7 @@
 from ..utils import check_random_state, check_array
 from ..utils._arpack import _init_arpack_v0
 from ..utils._param_validation import Interval, StrOptions
+from ..utils.fixes import eigh
```

Comment:
Should this be a private method? Historically, we have removed public functions from `utils.fixes` without deprecation, which I always found inconsistent.

Reply:
The advantage of having `eigh` public is that the body of the code does not change in the future. What would make sense is to have `fixes` private itself (i.e. `_fixes`).

I don't know what is best here.

Reply:
> The advantage of having eigh public is that the body of the code does not change in the future.

I don't understand this. Why would renaming it `_eigh` be a problem?

+1 for deprecating `sklearn.utils.fixes` in favor of `sklearn.utils._fixes` in a follow-up PR anyway.

Reply:
I agree with @ogrisel:
- let's use `eigh_` for this PR
- let's make `utils.fixes` private, I tried to find some past discussions about why it was not made private but I did not find anything in ~15 minutes. Maybe one reason I can think of is that we sometimes use `utils.fixes` in examples and that it would be a bit weird to use private stuff in examples. Personally I can live with using `utils._fixes` in examples when the alternative makes the example code a lot more complicated. Right now we have 4 examples using `utils.fixes`.

Reply:
I just pushed a commit doing that.

Reply:
Thanks, @lesteve for the push. Since we are few `utils.fixes`, it could be the right time to make it private and deprecate it (of course in another PR).